types/nick-thompson__elementary/package.json

```diff
@@ -0,0 +1,16 @@
+{
+    "private"        : true,
+    "dependencies"   : {
+        "@types/node": "*"
+    },
+    "devDependencies": {
+        "@nick-thompson/elementary": "^0.9.6",
+        "typescript"               : "~4.4.2",
+        "colors"                   : "^1.4.0"
+    },
+    "scripts"        : {
+        "build-test"    : "tsc --outDir ./build ./nick-thompson__elementary-tests.ts",
+        "test"          : "elementary -i 0 -o 2 ./build/nick-thompson__elementary-tests.js",
+        "build-and-test": "npm run build-test & npm run test"
+    }
```

Comment:
DT packages should only use DT's test infrastructure in the top-level package.json.